boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_shared.dts

```diff
@@ -0,0 +1,89 @@
+/ {
+	leds:	leds {
+		compatible = "gpio-leds";
+		rgb1_red: led_0 {
+			gpios = <&gpio0 7 GPIO_ACTIVE_HIGH>;
+			label = "0 LED_RGB_RED";
+		};
+		rgb1_green: led_1 {
+			gpios = <&gpio0 25 GPIO_ACTIVE_HIGH>;
+			label = "0 LED_RGB_GREEN";
+		};
+		rgb1_blue: led_2 {
+			gpios = <&gpio0 26 GPIO_ACTIVE_HIGH>;
+			label = "0 LED_RGB_BLUE";
+		};
+		rgb2_red: led_3 {
+			gpios = <&gpio0 28 GPIO_ACTIVE_HIGH>;
+			label = "1 LED_RGB_RED";
+		};
+		rgb2_green: led_4 {
+			gpios = <&gpio0 29 GPIO_ACTIVE_HIGH>;
+			label = "1 LED_RGB_GREEN";
+		};
+		rgb2_blue: led_5 {
+			gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
+			label = "1 LED_RGB_BLUE";
+		};
+		led1_blue: led_6 {
+			gpios = <&gpio0 31 GPIO_ACTIVE_HIGH>;
+			label = "2 LED_MONO_BLUE";
+		};
+		led2_green: led_7 {
+			gpios = <&gpio1 0 GPIO_ACTIVE_HIGH>;
+			label = "3 LED_MONO_GREEN";
+		};
+		led3_green: led_8 {
+			gpios = <&gpio1 1 GPIO_ACTIVE_HIGH>;
+			label = "4 LED_MONO_GREEN";
+		};
+
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		button_1_vol_dn: button_1_vol_dn {
+			gpios = <&gpio0 2 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button 1";
+		};
+		button_2_vol_up: button_2_vol_up {
+			gpios = <&gpio0 3 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button 2";
+		};
+		button3: button_3 {
+			gpios = <&gpio0 4 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button 3";
+		};
+		button4: button_4 {
+			gpios = <&gpio0 6 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button 4";
+		};
+		button5: button_5 {
+			gpios = <&gpio0 5 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+			label = "Push button 5";
+		};
+	};
+
+	nrf5340_audio_dk: nrf5340-audio-dk {
+		compatible = "nrf5340_audio_dk";
+		pmic-iset-out-gpios = <&gpio0 23 GPIO_ACTIVE_HIGH>;
+	};
```

Comment:
compatible here does not follow the guidelines, following e.g. actinius icarus board (similar case for sim select GPIOs), this could be `nordic-nrf5340-audio-dk`.

Reply:
Don't you mean `nordic,nrf5340-audio-dk`?

Reply:
In the case of actinius a dash was preferred because the sim selector was not a piece of hardware you can buy. Here, this is similar, so probably the name needs to be changed as well. `nordic-pmic-iset` maybe? See https://github.com/zephyrproject-rtos/zephyr/pull/47180#issuecomment-1212208423

Reply:
So to be clear @gmarull, which change are you suggesting here specifically? Change `nrf5340-audio-dk` to `nordic- nrf5340-audio-dk`? Is this consistent with all of our other DKs?

Reply:
We don't have anything similar on other DKs. The problem with this compatible is that it is not following the naming rules, so it needs to be fixed. The most similar precedent we have is the Actinius Icarus board, where they have a GPIO to indicate SIM selection, and it is configured as part of board init code. At that time, it was agreed to use the `vnd-name` scheme, so here I'd suggest `nordic-pmic-iset`. If the board has a PMIC IC (I thought it was removed according to NCS changes), then the proper solution is to create a regulator driver now that we have infrastructure in place.